http/src/routing/route.rs

```diff
@@ -1466,7 +1469,9 @@ impl<'a> Route<'a> {
             Self::InteractionCallback { interaction_id, .. } => {
                 Path::InteractionCallback(*interaction_id)
             }
-            Self::LeaveGuild { .. } => Path::UsersIdGuildsId,
+            Self::GetCurrentUserGuildMember { .. } | Self::LeaveGuild { .. } => {
```

Comment:
these aren't actually the same path, so there should be a new `Path` variant in `http-ratelimiting`